### PR TITLE
[CIS255-256] Add setup for `ChannelListController` and `ChannelEndpoints` tests

### PR DESCRIPTION
**Pull request by mantergo**
**In this PR:** 

- Add setup for `ChannelListController` and `ChannelEndpoints` tests

### CODE REVIEW

File: StreamChatClient_v3/Controllers/ChannelListController_Tests.swift [hunk 1]
```diff
@@ -81,7 +81,7 @@ class ChannelListController_Tests: StressTestCase {
         controller.startUpdating()
         
         // Assert the existing channel is loaded
-        XCTAssertEqual(controller.channels.map { $0.cid }, [cidMatchingQuery])
+        XCTAssertEqual(controller.channels.map(\.cid), [cidMatchingQuery])
```
Comment (VojtaStavik) [inline StreamChatClient_v3/Controllers/ChannelListController_Tests.swift:84]: This is not related to `Add setup for `ChannelListController` tests`

File: StreamChatClient_v3/Controllers/ChannelListController_Tests.swift [hunk 2]
```diff
@@ -140,7 +140,7 @@ class ChannelListController_Tests: StressTestCase {
         }
         
         // Assert the resulting value is updated
-        AssertAsync.willBeEqual(controller.channels.map { $0.cid }, [cid])
+        AssertAsync.willBeEqual(controller.channels.map(\.cid), [cid])
```
Comment (VojtaStavik) [inline StreamChatClient_v3/Controllers/ChannelListController_Tests.swift:143]: Neither is this

File: StreamChatClient_v3/Controllers/ChannelListController_Tests.swift [hunk 3]
```diff
@@ -223,16 +223,27 @@ class ChannelListController_Tests: StressTestCase {
...
+                                                                        return self.channelUpdater!
+                                                                    },
                                                                     changeAggregatorBuilder: { [unowned self] in
-        self.changeAggregator = ChangeAggregator(itemCreator: $0)
-        return self.changeAggregator!
-        })
+                                                                        self.changeAggregator = ChangeAggregator(itemCreator: $0)
+                                                                        return self.changeAggregator!
+                                                                    })
 }
```
Comment (VojtaStavik) [inline StreamChatClient_v3/Controllers/ChannelListController_Tests.swift:247]: The formatter went really crazy here 🤦‍♂️ We need to something about it soon.

File: StreamChatClient_v3/Controllers/ChannelListController_Tests.swift (hunk 1 above)
Comment (mantergo) [reply StreamChatClient_v3/Controllers/ChannelListController_Tests.swift:84]: It is SwiftFormat change

Comment (VojtaStavik) [reply StreamChatClient_v3/Controllers/ChannelListController_Tests.swift:84]: You probably ran it manually, because it's set only as a pre-commit hook. It doesn't format code that's not staged.

File: StreamChatClient_v3/Controllers/ChannelListController_Tests.swift (hunk 3 above)
Comment (buh) [reply StreamChatClient_v3/Controllers/ChannelListController_Tests.swift:247]: This is actually correct behaviour. The same as in Xcode.
